Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	go build
+
+test: build
+	go test -v -count=1 ./testing
+
+PHONY: build test
```

Comment:
I think it should be `.PHONY`. I'm not sure if it works without the starting dot.